src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < 3; i++) {
```

Comment:
Think how to avoid using 3 for loops. 
Also, 6 is a magic number, make it constant

Reply:
Do I need to make 6 as const in main method to test if everything is working?